runtime/auth/signing/aws-signing-crt/jvm/test/aws/smithy/kotlin/runtime/auth/signing/awssigning/crt/CrtSigningSuiteTest.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+package aws.smithy.kotlin.runtime.auth.signing.awssigning.crt
+
+import aws.smithy.kotlin.runtime.auth.signing.awssigning.common.AwsSigner
+import aws.smithy.kotlin.runtime.auth.signing.awssigning.tests.SigningSuiteTestBase
+
+class CrtSigningSuiteTest : SigningSuiteTestBase() {
+    override val signer: AwsSigner = CrtAwsSigner
+
+    override val disabledTests = setOf(
+        // ktor-http-cio parser doesn't support parsing multiline headers since they are deprecated in RFC7230
```

Comment:
Just rechecked: they all still fail. How could we support these if Ktor's parser can't handle them?

Reply:
A different parser perhaps? Or even our own? We don't need a full fledged HTTP message parser, just one that works for the tests (and these test files are overall pretty simple). This can be a separate PR though